microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -1036,6 +1036,32 @@ See also:
 * [ICoreWebView2_2::WebResourceResponseReceived event (add](/microsoft-edge/webview2/reference/win32/icorewebview2_2#add_webresourceresponsereceived), [remove)](/microsoft-edge/webview2/reference/win32/icorewebview2_2#remove_webresourceresponsereceived)
    * [ICoreWebView2WebResourceResponseReceivedEventArgs interface](/microsoft-edge/webview2/reference/win32/icorewebview2webresourceresponsereceivedeventargs)
 
+The `CustomSchemeRegistration` allows registration of custom schemes in WebView2 so that app can handle the `WebResourceRequested` event for requests to those custom scheme URLs and navigate the WebView2 to it.
```

Comment:
```suggestion
The `CustomSchemeRegistration` allows registration of custom schemes in WebView2 so that app can handle the `WebResourceRequested` event for requests to those custom scheme URLs and navigate the WebView2 to it.

---
```
tab set terminator for the above tab-set

Reply:
@nishitha-burman - we need to add an h4 heading here, right?  or should be h2 big heading?  Is this wording ok, 
```
#### Custom scheme registration
```